packages/cli-kit/src/private/node/ui/components/SelectInput.tsx

```diff
@@ -114,7 +114,9 @@ function Item<T>({
 
       <Box key={index} marginLeft={hasAnyGroup ? 3 : 0}>
         <Box marginRight={2}>{isSelected ? <Text color="cyan">{`>`}</Text> : <Text> </Text>}</Box>
-        <Text color={labelColor}>{showKey ? `(${item.key}) ${label}` : label}</Text>
+        <Text wrap="end" color={labelColor}>
```

Comment:
It seems to be doing what `wrap="wrap"` is supposed to do, which is wrap the text when it's at the end of the component - I see here that `wrap="wrap"` is default for `Text` component, but the text only wrapped after I specified `wrap="end"` here

Reply:
weird though, I can't see where `end` is being called here, but maybe there's some bug related to their `wrap` code that makes it so that setting `wrap="end"` actually doesn't default anything and somehow fixes our issues?

https://github.com/vadimdemedes/ink/blob/7fedab06770eb6199aafe06163f9c47884b88f58/src/wrap-text.ts#L42